FishBun/src/main/java/com/sangcomz/fishbun/ui/album/AlbumController.java

```diff
@@ -78,11 +78,16 @@ private String getAllMediaThumbnailsPath(ContentResolver resolver, long id) {
                 Cursor cursor = resolver.query(images, null,
                         selection, selectionArgs, sort);
                 if (cursor != null && cursor.moveToNext()) {
+                    //int imgId = c.getInt(c.getColumnIndex(MediaStore.MediaColumns._ID));
```

Comment:
I have raised new pull request, these code has been merged as part of "replace file path with Uri"